proxy.go

```diff
@@ -152,9 +154,20 @@ func NewProxyFromConfig(logger *logrus.Logger, conf ProxyConfig) (p Proxy, err e
 
 	p.TraceClient = trace.DefaultClient
 	if conf.SsfDestinationAddress != "" {
+		stats, err := statsd.NewBuffered(conf.StatsAddress, 4096)
+		if err != nil {
+			return p, err
+		}
+		stats.Namespace = "veneur_proxy."
+		format := "ssf_format:packet"
+		if strings.HasPrefix(conf.SsfDestinationAddress, "unix://") {
+			format = "ssf_format:framed"
+		}
+
 		p.TraceClient, err = trace.NewClient(conf.SsfDestinationAddress,
 			trace.Buffered,
 			trace.FlushInterval(3*time.Second),
+			trace.ReportStatistics(stats, 1*time.Second, []string{format}),
```

Comment:
Oh, I see. The client does the work but needs a statsd client. I think that the appearance of a metrics client in a metrics server (with your recent patches removing them!) just had me a bit turned around.